src/ui/components/Events/ReplayInfo.tsx

```diff
@@ -90,6 +91,40 @@ function ReplayInfo({ setModal }: PropsFromRedux) {
             </Row>
           ) : null}
         </div>
+        <div className="group">
```

Comment:
Will make sure to kick off a discussion tomorrow around how the experience of debugging a replay should look like if you're specifically debugging replay of a test. For now, this should do the trick.